src/components/Sidebar/NavContent.jsx

```diff
@@ -95,6 +96,7 @@ function NavContent() {
 
             {address && (
               <div className="wallet-link">
+                <Davatar size={20} address={address} style={{ marginRight: 8 }} generatedAvatarType="jazzicon" />
```

Comment:
blockies styling needs a quick adjustment to pick up this same `marginRight: 8`
![image](https://user-images.githubusercontent.com/80423742/147484651-46205626-1c66-41bf-b3ac-03165aea89dc.png)

Reply:
@appleseed-iii this is fixed now 

![image](https://user-images.githubusercontent.com/69431456/147486152-b628b1c9-5744-4af1-9b0b-811d23b89d90.png)

Reply:
thanks ser